alembic/versions/2d2af8a9c996_upgrade_change_log.py

```diff
@@ -18,7 +18,7 @@
 
 def upgrade():
     op.execute('ALTER TABLE change_log ADD COLUMN big_id BIGINT;')
-    op.execute('CREATE FUNCTION set_new_id() RETURNS TRIGGER AS\n'\
+    op.execute('CREATE OR REPLACE FUNCTION set_new_id() RETURNS TRIGGER AS\n'\
```

Comment:
Added this just so this doesn't error out if the function already exists for some reason.